src/components/EditDeletePostModal.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { deletePost } from "../features/postSlice";
+import { useDispatch, useSelector } from "react-redux/es/exports";
```

Comment:
Do keep the import from "react-redux" instead of the VS Code provided file path

Reply:
sure